flink-clients/src/test/java/org/apache/flink/client/program/rest/RestClusterClientTest.java

```diff
@@ -595,35 +458,6 @@ public void testRetriableSendOperationIfConnectionErrorOrServiceUnavailable() th
 		}
 	}
 
-	@Test
-	public void testJobSubmissionFailureThrowsProgramInvocationException() throws Exception {
```

Comment:
Why is this test being removed?

Reply:
This test tests `ClientUtils.submit` which has been removed in this pr.

Reply:
similar to below, we may want to adjust the test to instead use the new APIs and check the exception behavior.